_1_basics/src/main/java/code/_4_student_effort/Pairs.java

```diff
@@ -0,0 +1,36 @@
+package code._4_student_effort;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Pairs {
+
+    private static int pairs = 0;
+
+    public static void main(String[] args) {
+        // modify the input here
+        int[] input = {5, 9, -5, 7, -5};
+
+        // using a list to simplify the process of eliminating pairs
+        List<Integer> num = new ArrayList<>();
+        for(int i = 0; i < input.length; i++)
+            num.add(input[i]);
+
+        for(int i = 0; i < num.size(); i++) {
+            for(int j = i + 1; j < num.size(); j++) {
+
+                if(num.get(i) + num.get(j) == 0) {
```

Comment:
ce complexitate ciclomatica are implementarea ta ? in termen de big O notation ...

si ce modificari am putea face a.i. sa reducem complexitatea ciclomatica ?